src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.Linux.cs

```diff
@@ -462,7 +452,14 @@ private void InitializeSysFS()
                     return;
                 }
 
-                _sysFSDriver = new SysFsDriver();
+                try
+                {
+                    _sysFSDriver = new LibGpiodDriver(0);
+                }
+                catch (PlatformNotSupportedException)
```

Comment:
Is it PlatformNotSupported what we throw? I thought it was FileNotFound when we try to PInvoke and don't find it.

Reply:
Nvm, I just saw that we catch the dllNotFound on the constructor and then throw a PNSE